crates/sp-domains-fraud-proof/src/lib.rs

```diff
@@ -150,6 +150,7 @@ impl PassBy for StatelessDomainRuntimeCall {
 
 sp_api::decl_runtime_apis! {
     /// API necessary for fraud proof.
+    #[api_version(2)]
```

Comment:
It might be helpful to add a comment saying what changed in the API version, because it's not clear from the  `decl_runtime_apis` block.